scripts/cnode-helper-scripts/startupLogMonitor.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+#shellcheck disable=SC2086,SC2001,SC2154
+#shellcheck source=/dev/null
+
+STARTUPLOG_DIR="/opt/cardano/cnode/guild-db/startuplog"
+STARTUPLOG_DB="${STARTUPLOG_DIR}/startuplog.db"
+logfile="/opt/cardano/cnode/logs/node0.json"
```

Comment:
IMO - while user variables being made available are fine, the locations should already be available if sourcing `env` file , going by the usual convention - user variables get defaults assigned automatically, unless variable is customised

Reply:
Additional commit which includes sourcing via `env` w/ **OFFLINE_MODE** to bypass `node hasnt completed startup` messages.